doc/en/user/source/services/wms/get_legend_graphic/index.rst

```diff
@@ -141,6 +142,7 @@ The producer for the raster legend will make use of this elements in order to bu
     - the height of each row is set to the maximum height of the single elements
     - the width of each row is set to the sum of the width of the various elements plus the various paddings
     - **dx,dy** the spaces between elements and rows are set to the 15% of the requested width and height. Notice that **dy** is ignored for the colormaps of type **ramp** since they must create a continuous color strip.
+    - **absoluteMargins** true/false, if true dx is considered as a fixed number of pixels, instead of a percentage of the width.

```

Comment:
Maybe it should be clear that this option only affects rasters legends and that it may be combined with the dx option to get the desirable margin.